bundles/org.openhab.binding.nanoleaf/README.md

```diff
@@ -94,6 +93,8 @@ Compare the following output with the right picture at the beginning of the arti
                                     41451                                     
 
 ```
+
+For other canvases, use the Layout channel on the controller to get a picture of the layout with the thing IDs.
```

Comment:
Thanks! I have tried to add some more documentation and a picture of my layout. I didn't add anything from the OpenHAB UI so that the picture stays the same even though the UI may change.